webui.dev/js/marshaller.js

```
@@ -2,6 +2,33 @@
+function checkAndTriggerActionFromQueryParam () {
+  const params = getQueryParams()
+
+  const action = params.get('action')
+  if (action && window.actionButtons) {
+    // Look for an action button with matching title
+    const actionButton = window.actionButtons[action]
+
+    if (actionButton) {
+      // Only trigger actions that have arguments
+      const jsonButton = window.actionButtonsJson[action]
+      if (jsonButton && jsonButton.arguments && jsonButton.arguments.length > 0) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)